apps/admin_app/lib/admin_app_web/views/general_settings_view.ex

```diff
@@ -2,7 +2,13 @@ defmodule AdminAppWeb.GeneralSettingsView do
   use AdminAppWeb, :view
   alias Snitch.Data.Model.ProductBrand
 
+  @currencies ["USD", "INR", "GDP", "EUR"]
```

Comment:
I was thinking if any other app in the umbrella would be needing the list of currencies. If it does then it will be required to move to some other place. So how about keeping it at a place in app that can expose this to all other apps.

Reply:
I think `core` should contain list of valid currencies